### ISSUE TITLE
not supported for cordova android 7.0.0

### ISSUE THREAD

**Issue by imadarif** [Telerik-Verified-Plugins/BarcodeScanner#53]
Plugin doesn't support this project's cordova-android version. cordova-android: 7.0.0, failed version requirement: 
      <6.3.0
Skipping 'cordova-plugin-compat' for android

* What went wrong:
Could not resolve all files for configuration ':app:debugCompileClasspath'.
> Could not find :barcodescanner:.

not working on cordova-android 7.0.0

**Comment by Fever905** [comment]
Same issue here.

**Comment by nadyaA** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Feature 2](https://trello.com/c/F2dEENdK/2-feature-2)

**Comment by Fever905** [comment]
Is this ever going to be resolved?

**Comment by BrianPYpsilon** [comment]
I have the same problem. I can't build for Android now because another plugin needs PhoneGap > 7.1.0

**Comment by Fever905** [comment]
Mine has been working now for a while.  Did you get your build issues sorted out?

-
Michael DeMutis
Web Applications Developer
Wearcheck Canada Inc.
Burlington, ON  Canada
Phone: (905) 569-8600 x 0723
Direct:  (289) 291-0723
email:  michael.demutis@wearcheck.com<mailto:michael.demutis@wearcheck.com>
web:  www.wearcheck.ca<http://www.wearcheck.ca/>  www.lubrigard.com<http://www.lubrigard.com/>
THE OIL SPEAKS. WEARCHECK LISTENS. YOU SAVE.

**Comment by BrianPYpsilon** [comment]
Yes, thanks. I just removed the explicit version number from config.xml. This then caused an issue with the iOS build in that, although the build worked, the NSCameraUsageDescription was not written to the info.plist for iOS. This was fixed by adding an explicit edit-config statement:
    <edit-config target="NSCameraUsageDescription" file="*-Info.plist" mode="overwrite">
        <string>Used for bar code scanning only</string>
    </edit-config><!--<plugin name="cordova-plugin-device-orientation" spec="1.0.5">

So everything is fine now, thanks
